back/app/services/side_fx_participation_context_service.rb

```diff
@@ -22,7 +22,23 @@ def before_update(pc, _user)
     end
   end
 
-  def after_update(pc, _user)
+  def after_update(pc, user)
```

Comment:
Appreciate that you didn't change this, but as a matter of style I don't think we should have abbreviated variable names - unless within a single line map {} maybe

Reply:
Shall I rename it to "context"?